src/reflect/scala/reflect/internal/util/ScalaClassLoader.scala

```diff
@@ -43,7 +43,7 @@ final class RichClassLoader(private val self: JClassLoader) extends AnyVal {
   def tryToInitializeClass[T <: AnyRef](path: String): Option[Class[T]] = tryClass(path, initialize = true)
 
   private def tryClass[T <: AnyRef](path: String, initialize: Boolean): Option[Class[T]] =
-    catching(classOf[ClassNotFoundException], classOf[SecurityException]) opt
+    catching(classOf[ClassFormatError], classOf[ClassNotFoundException], classOf[SecurityException]) opt
```

Comment:
Lukas and I were a bit curious about this part — it seems it isn't necessary for the test? What was the motivation?

Reply:
You'll see it without the fix. The method expects failures loading the class, but I did not make an exhaustive research.